### PR TITLE
Adding a 'return' button for ReminderCommand.

### PR DESCRIPTION
**Pull request by VitorBlog**
Idk if it's working because my knowledge of the loritta api is very basic.

### CODE REVIEW

File: loritta-discord/src/main/java/com/mrpowergamerbr/loritta/commands/vanilla/utils/LembrarCommand.kt [hunk 1]
```diff
@@ -171,13 +171,13 @@ class LembrarCommand : AbstractCommand("remindme", listOf("lembre", "remind", "l
...
 				}
-
-				context.reply(
-                        LorittaReply(
-                                locale["${LOCALE_PREFIX}.reminderRemoved"],
-                                "\uD83D\uDDD1"
-                        )
-				)
+				
+				val message = context.sendMessage("locale["${LOCALE_PREFIX}.reminderRemoved"]")
```
Comment (SrGaabriel) [inline loritta-discord/src/main/java/com/mrpowergamerbr/loritta/commands/vanilla/utils/LembrarCommand.kt:175]:
```suggestion
				val message = context.sendMessage(locale["${LOCALE_PREFIX}.reminderRemoved"])
```

**Review by SrGaabriel — CHANGES_REQUESTED** [review]
This is why the build is failing.

**Comment by MrPowerGamerBR** [comment]
You could try selfhosting Loritta and see if it works or not

**Comment by VitorBlog** [comment]
Tested
loretta

**Comment by VitorBlog** [comment]
Better GIF
lorota

**Comment by VitorBlog** [comment]
When testing, I noticed that the reminder doesn't translate because the 'remindme.yml' file only exists on default locale.

**Comment by MrPowerGamerBR** [comment]
LGTM

**Pull request merged**